### PR TITLE
People

### PR DESCRIPTION
**Pull request by jeremyfourna**
Separate the list of people into their own `const` until we have proper DB

### CODE REVIEW

**Comment by jeremyfourna** [comment]
Should be merge after #2

File: data.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
+/* jshint esversion: 6 */
+
+const restaurants = [{
+  name: 'W Imbiss',
```
Comment (Chemaclass) [inline data.js]: I think we should placed the name of the restaurant as const as well, so we use const below for the restaurants from the "people". Doing that we get rid of raw-strings. Actually, I see that you are using the name of the restaurant as ID... what if the name of the restaurant changes? ...
I'm saying this from the perspective of from where are you doing the relation between the restaurant and the people.

**Comment by Chemaclass** [comment]
I think we should use proper ids (int) instead of the restaurant.names for the relations between people and restaurants.

File: data.js (hunk 1 above)
Comment (jeremyfourna) [reply data.js]: OK will add that

File: index.html [hunk 2]
```diff
@@ -12,6 +12,14 @@
     <h1>Easy Lunch</h1>
     <div id="app"></div>
     <script type="text/javascript" src="index.js"></script>
+    <script type="text/javascript" src="restaurants.js"></script>
+    <script type="text/javascript" src="people.js"></script>
+    <script type="text/javascript">
+    const people = [jeremy, andrey, rahul];
```
Comment (RahulReddy75) [inline index.html]: why no chema and all the others?

**Review by RahulReddy75 — APPROVED** [review]
look comment and merge.

File: index.html (hunk 2 above)
Comment (Chemaclass) [reply index.html]: exactly!

File: index.html [hunk 3]
```diff
@@ -12,6 +12,14 @@
...
+
+    console.log(randomRestaurantForToday(restaurants, people));
```
Comment (Chemaclass) [inline index.html]: Display the result on the page itself, not on console.log!

File: people.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
+/* jshint esversion: 6 */
+
+const jeremy = {
+  name: 'Jeremy',
+  favCuisine: ['Lebanese', 'Indian', 'French', 'Japanese'],
```
Comment (Chemaclass) [inline people.js:5]: I would create const for favCuisine instead of raw-strings

File: people.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  }, {
+    id: 6,
+    score: -3
+  }],
+  lunchThisWeek: []
+};
+
+const rahul = {
+  name: 'Rahul',
+  favCuisine: [],
```
Comment (Chemaclass) [inline people.js:43]: @RahulReddy75 has no favCuisine?

File: people.js [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+  }, {
+    id: 1,
+    score: 1
+  }],
+  lunchThisWeek: []
+};
+
+const chema = {
+  name: 'Chema',
+  favCuisine: ['German'],
```
Comment (Chemaclass) [inline people.js]: I also love Vietnamiese!

File: people.js [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+  restaurants: [{
+    id: 6,
+    score: 3
+  }],
+  lunchThisWeek: []
+};
+
+const piotr = {
+  name: 'Piotr',
+  favCuisine: [],
```
Comment (Chemaclass) [inline people.js:78]: We need to ask Piotr for his favs ones!

File: people.js (hunk 4 above)
Comment (jeremyfourna) [reply people.js:5]: we will need in this case an entity cuisine type ?

File: index.html (hunk 2 above)
Comment (jeremyfourna) [reply index.html]: I added chema

File: index.html (hunk 3 above)
Comment (jeremyfourna) [reply index.html]: Will do it in the next PR

File: people.js (hunk 4 above)
Comment (jeremyfourna) [reply people.js:5]: Will do in a next PR

File: people.js (hunk 5 above)
Comment (jeremyfourna) [reply people.js:43]: Indian maybe ? @RahulReddy75

File: people.js (hunk 4 above)
Comment (Chemaclass) [reply people.js:5]: Yes, I would say we need the cuisine entity as well. So we have the IDs, and we don't care about the name itself for the relations ;)

File: people.js (hunk 6 above)
Comment (jeremyfourna) [reply people.js]: Done

File: people.js (hunk 4 above)
Comment (Chemaclass) [reply people.js:5]: Either you create the type Cuisine(id,name) or you create some const for the Cuisine name... up to you.

**Pull request merged**